data/policies/conditional.cas

```diff
@@ -17,6 +18,18 @@ domain bar {
 		allow(this, foo, file, getattr);
 	}
 
+	if (my_tunable && my_tunable_2) {
+		allow(this, foo, file, setattr);
+	}
+
+	if (my_tunable || my_tunable_2) {
+		allow(this, foo, file, open);
+	}
```

Comment:
If the parser can handle it, it would be a good idea to test stuff like (!(x && y)) and (x && !y)